content/_index.ca.md

```diff
@@ -36,8 +36,7 @@ excursionistes i ciclistes:
 - Línies de contorn, perfils d'elevació, pics i pendents
 - Navegació pas a pas a peu, amb bicicleta i amb cotxe amb guia de veu
 - Cerques ràpides al mapa, sense connexió
-- Els marcadors s'exporten i importen en els formats KML/KMZ (GPX està
-  [planificat][gpx_issue])
+- Els marcadors s'exporten i importen en els formats KML/KMZ/GPX
```

Comment:
@rtsisyk  We only support GPX import now, not export. Do you have any ideas on how to change these texts?

Reply:
This is sad... What is about:

> Bookmarks export and import in KML/KMZ/GPX formats

=>

> Saved places and tracks in KML/KMZ/GPX formats